ui/package.json

```diff
@@ -68,6 +68,7 @@
     "react-onclickoutside": "^6.7.1",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
+    "react-router-hash-link": "^2.4.3",
```

Comment:
React doesn't seem to be able to follow `#foo` links without this.  All my searching led me back to this... which seems to work fine.